src/ray/protobuf/ray_client.proto

```diff
@@ -114,7 +114,7 @@ message PutResponse {
 // Requests data from the server.
 message GetRequest {
   // The reference ID for the requested object data
-  bytes id = 1;
```

Comment:
Is it possible to change this field name to plural?

Reply:
Good suggestion, updated.

Reply:
Wait, @mwtian I strongly prefer following the convention of not modifying existing proto fields. Can we mark `id` as deprecated and add a new field `ids`

Reply:
Also make sure to update the PROTO_VERSION const!

Reply:
This should be fine given that we updated the protocol. In general I usually prefer something like:
```
bytes id = 1 [deprecated = true];
float timeout = 2;
bool asynchronous = 3;
repeated bytes ids = 4;

```